LEAF_Request_Portal/utils/db_procedures/data_stringReplace.php

```diff
@@ -0,0 +1,28 @@
+<?php
+ini_set('display_errors', 0); 
+
+
+define("LF", "\n");
+include '../db_mysql.php';
+include '../db_config.php';
+
+$debug = false;
+$db_config = new DB_Config();
+
+$db = new DB($db_config->dbHost, $db_config->dbUser, $db_config->dbPass, $db_config->dbName);
```

Comment:
Defined but not used.

Reply:
i should remove this file since it's replaced by procedure_ReplaceErrorMsg.sql